plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupProvider.test.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createLogger, transports } from 'winston';
+import { OktaGroupProvider } from './OktaGroupProvider';
+import { ConfigReader } from '@backstage/config';
+import { EntityProviderConnection } from '@backstage/plugin-catalog-backend';
+import { MockOktaCollection } from '../test-utls';
+
+let listGroups: () => MockOktaCollection = () => {
+  return new MockOktaCollection([]);
+};
+
+jest.mock('@okta/okta-sdk-nodejs', () => {
+  return {
+    Client: jest.fn().mockImplementation(() => {
+      return {
+        listGroups,
+      };
+    }),
+  };
+});
+
+const logger = createLogger({
+  transports: [new transports.Console({ silent: true })],
+});
```

Comment:
there is a voidlogger in the backend-common package that is being used most of the time in the tests.

```suggestion
import { getVoidLogger } from '@backstage/backend-common';
const logger = getVoidLogger()
```